webapp/src/ts/pipes/contacts.pipe.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'filterReports'
```

Comment:
It's actually better to use a pipe because it is only called when the input value changes. I have tested it and it is not called unnecessary on every change detection.

Reply:
Awesome, so it only runs when the value changes 👍 Thank you!